server/appenders/cassandra/appender/src/main/java/org/kaaproject/kaa/server/appenders/cassandra/appender/CassandraLogAppender.java

```diff
@@ -50,8 +53,20 @@
     private boolean closed = false;
     private CassandraExecuteRequestType executeRequestType;
 
+    private final ScheduledExecutorService scheduler = Executors.newScheduledThreadPool(1);
+    private final AtomicLong inputLogCount = new AtomicLong();
+    private final AtomicLong cassandraLogCount = new AtomicLong();
+
     public CassandraLogAppender() {
         super(CassandraConfig.class);
+        scheduler.scheduleWithFixedDelay(new Runnable() {
```

Comment:
I believe this changes are related to performance test and should not be merged to master